vllm/worker/hpu_model_runner.py

```diff
@@ -727,6 +726,15 @@ def __init__(
         # For delayed sampling
         self.cached_step_inputs: List[
             ModelInputForHPUWithSamplingMetadata] = []
+        self.profile_execute_model_prompt = os.environ.get(
```

Comment:
Are those env vars necessary? Why `VLLM_PT_PROFILE` cannot be used for profiling?

Reply:
removed profiling in this PR. The 'VLLM_PT_PROFILE' produce dummy input which can't fully test MOE which makes profiling faster.